sample/ApiSamples/Program.cs

```diff
@@ -19,6 +19,8 @@ internal class Program
 	{
 		private static void Main(string[] args)
 		{
+			Environment.SetEnvironmentVariable("ELASTIC_APM_LOG_LEVEL", "Trace");
```

Comment:
Yes, I left it in on purpose since this is one of my go-to samples when trying something out and I usually prefer verbose log output in those cases.
(I should have added a comment about this, sorry).